spec/elasticsearch_cluster_spec_helper.rb

```diff
@@ -9,45 +9,37 @@ module ElasticsearchClusterSpecHelper
 
   ELASTICSEARCH_TEST_INDEX     = "test_example_index"
   ELASTICSEARCH_TEST_LOCALHOST = ENV["ELASTICSEARCH_TEST_HOST"].presence || "127.0.0.1"
-  ELASTICSEARCH_TEST_PORT      = ENV["ELASTICSEARCH_TEST_PORT"]&.to_i || 9250
-
-  # If using Minitest, use Minitest::Hooks.
-
-  # `export ELASTICSEARCH_TEST_PORT=9200` to run tests against your local, running Elasticsearch
-
-  # If you run into issues with Webmock, make sure to allow local requests
-  # WebMock.disable_net_connect!(allow_localhost: true) # allow local requests in setup
-  # WebMock.disable_net_connect! # disallow all requests in teardown
-
-  CLUSTER_COMMANDS =
-    if ENV["SEMAPHORE_CI_ELASTICSEARCH"].present?
-      { command: "/usr/share/elasticsearch/bin/elasticsearch", es_params: "-E path.conf=/etc/elasticsearch/" }
-    else
-      {}
-    end.freeze
+  ELASTICSEARCH_TEST_PORT      = ENV["ELASTICSEARCH_TEST_PORT"]&.to_i || 9200
 
   def clear_and_create_index(index: ELASTICSEARCH_TEST_INDEX)
     if @elasticsearch_test_client&.indices&.exists?(index: index)
-      @elasticsearch_test_client.indices.delete(index: index)
-    end
-
-    unless @elasticsearch_test_client&.indices&.exists?(index: index)
-      @elasticsearch_test_client.indices.create(index: index)
+      @elasticsearch_test_client.indices.refresh(index: index)
+      @elasticsearch_test_client.delete_by_query(index: index, body: {
+        query: {
```

Comment:
This is faster for local development than deleting and recreating the entire index.